src/index.es

```diff
@@ -1,3 +1,79 @@
-import React from 'react';
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import clsx from 'clsx';
+import addMonths from 'date-fns/addMonths';
+import { getMonth, getMonthLabel } from './utils';
+import MonthView from './month_view';
 
-export default () => <h1>Hello World</h1>;
+const Controls = ({ theme, label, onNext, onPrevious }) => (
+  <div className={theme.controls}>
+    <i onClick={onPrevious} className={theme.controlsPrevious} />
+    <span className={theme.controlsLabel}>{label}</span>
+    <i onClick={onNext} className={theme.controlsNext} />
+  </div>
+);
+
+const MonthCalendar = ({
+  className: passedClassName,
+  theme,
+
+  firstDay,
+  weekdayShortLabels,
+  monthLabels,
+
+  selected,
+  onChange,
+  minDate,
+  maxDate,
+}) => {
+  const locale = { firstDay, weekdayShortLabels, monthLabels };
+  const [month, setMonth] = useState(null);
+
+  const visibleMonth = month || getMonth(selected || new Date());
```

Comment:
i don't think we need to support resetting default selection at the time, and this doesn't fully support it anyway